ledger/metrics/src/main/scala/com/daml/metrics/ParticipantMetrics.scala

```diff
@@ -0,0 +1,686 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.metrics
+
+import java.time.Instant
+
+import com.codahale.metrics.MetricRegistry.MetricSupplier
+import com.codahale.metrics._
+
+final class ParticipantMetrics(override val registry: MetricRegistry) extends Metrics(registry) {
```

Comment:
It would have been a significant sanity improvement to leave this named `Metrics` and place it in the `participant` subpackage, *or* to have every import do `ParticipantMetrics => Metrics` and leave all the existing references alone. We aren't working with Java.

Reply:
...which would avoid, e.g., the conflict in ParallelIndexerFactory as of 70cca4b.

Reply:
Didn't though of just putting it into a different package, thanks for the idea @S11001001 :)!